test/endsWith.js

```diff
@@ -0,0 +1,20 @@
+var R = require('..');
+var eq = require('./shared/eq');
+
+describe('startsWith', function() {
+  it('should return true when a string ends with the provided value', function() {
+    eq(R.endsWith('c', 'abc'), true);
+  });
+
+  it('should return false when a string does not end with the provided value', function() {
+    eq(R.endsWith('b', 'abc'), false);
+  });
+
+  it('should return true when an array ends with the provided value', function() {
+    eq(R.endsWith(['c'], ['a', 'b', 'c']), true);
+  });
+
+  it('should return false when an array does not end with the provided value', function() {
+    eq(R.endsWith(['b'], ['a', 'b', 'c']), false);
+  });
+});
```

Comment:
I would prefer it if at least one test demonstrated that the input does not have to be unary:  `endsWith(
'ology', 'physics'); //=> false`, or `endsWith([3, 0, 9], [8, 6, 7, 5, 3, 0, 0])' //=> true`,  This could be an additional test, or a change to one that's there.